exercises/affine-cipher/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+/// While the problem description indicates a return status of 1 should be returned on errors,
+/// it is much more common to return a `Result`, so we provide an error type for the result here.
```

Comment:
Your reasoning here is precisely correct, but you should be aware of https://github.com/exercism/problem-specifications/pull/1614. At your discretion, you can choose to wait for that issue to be resolved before moving forward with this PR; just let me know. If you'd prefer to get this merged as quickly as possible, that is acceptable.

Reply:
I can wait, and I'll update the phrasing when the specification's description.md gets updated.